### PR TITLE
runfix: Restore identity resetting

### PR DESCRIPTION
**Pull request by atomrc**
Will leave a way for QA to break the identity in both Cryptobox and CoreCrypto env

### CODE REVIEW

File: src/script/util/DebugUtil.ts [hunk 1]
```diff
@@ -110,7 +110,8 @@ export class DebugUtil {
   }
 
   async resetIdentity(): Promise<void> {
-    this.core['coreCryptoClient'].wipe();
+    const proteusService = this.core.service!.proteus;
+    await proteusService['cryptoClient'].debugResetIdentity();
   }
```
Comment (tlebon) [inline src/script/util/DebugUtil.ts:115]: should there be a proteus and coreCrypto version of this function?

Comment (atomrc) [reply src/script/util/DebugUtil.ts:115]: That's the beauty of it, the `CryptoClient` is a generic interface that wrap both CoreCrypto and Cryptobox, no need for the webapp to know what is underneath :)

Comment (tlebon) [reply src/script/util/DebugUtil.ts:115]: ah but this reset identity function *is* only for proteus then. i suppose the MLS equivalent exists elsewhere.

Comment (atomrc) [reply src/script/util/DebugUtil.ts:115]: If coreCrypto is used (which is the case for MLS) MLS is handled as well in this method 👌

**Pull request merged**